products/statement-generator/public/locales/en/translation.json

```diff
@@ -75,13 +92,12 @@
     "previewing": "Previewing",
     "introduce_yourself": "Introduce yourself",
     "involvement": "Involvement",
-    "job": "Employment",
```

Comment:
also I think lines 78 and onward is overriding changes from dev unrelated to what you're updating here.

Reply:
Good catch!